content/millennium/r4/scheduling/appointment.md

```diff
@@ -146,6 +157,8 @@ _Implementation Notes_
 
 * This implementation follows the [JSON PATCH](https://tools.ietf.org/html/rfc6902) spec.
 * Only operations on the paths listed below are supported.
+* Video visit link patch requests will return a version code of the format "<previous_version>-1" (i.e. a preiously unpatched appointment returns "1-1")
```

Comment:
technically we really don't need to call this out since version ids are opaque

Reply:
https://github.com/cerner/fhir.cerner.com/pull/294/commits/2662d3989c06b836a807167951a85a63068e21b5#diff-1d7df8ceb9f0ae8471323516e037c662L160